pkg/collector/corechecks/snmp/checkconfig/config.go

```diff
@@ -33,6 +33,8 @@ const subnetTagPrefix = "autodiscovery_subnet"
 // - snmp-net uses 10
 const DefaultBulkMaxRepetitions = uint32(10)
 
+var uptimeMetricConfig = MetricsConfig{Symbol: SymbolConfig{OID: "1.3.6.1.2.1.1.3.0", Name: "sysUpTimeInstance"}}
```

Comment:
can this be `const` ?

Reply:
It can't, using `const` here will lead to:

```
pkg/collector/corechecks/snmp/checkconfig/config.go:36:7: const initializer MetricsConfig literal is not a constant
```

It's because `MetricsConfig` is a struct and can be changed.